### DISCUSSION TITLE
기술 스택, 댓글

### DISCUSSION THREAD

**Discussion by qkdl60** [prgrms-fe-devcourse/FEDC4_CUCUMIS_Pocojang#50] [General]
- Developerlist Page에서  각 카드에 있는 기술스택을 몇 개까지 보여줘야 할까요? 
 -댓글 길이 제한이 필요할까요, 그리고 댓글 리스트에서 어디까지 보여주는게 좋을 까요?

**Comment by kutta97** [comment]
따로 해당 아젠다로 회의 시간을 잡아보시죠!
